flume/conf/README.md

```diff
@@ -0,0 +1,54 @@
+# Cygnus configuration Procedure
+
+From version 0.6.0 Cygnus can start multiple instances by renaming and configuring some of 
```

Comment:
Fix in [e46fe68](https://github.com/telefonicaid/fiware-connectors/commit/e46fe6894611cc927e1916e003da3d9a9b39a8ef)